docs/contrib/issues.md

```diff
@@ -0,0 +1,19 @@
+# Reporting issues
+
+If you notice an issue (a typo, a grammar glitch, or missing or incorrect information), here's how you can let us know:
```

Comment:
I would replace `or missing or incorrect information` with `missing or factually incorrect information`.

Reply:
Would you mind using [the suggestion feature](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request) to make your suggestions? I would like to try something out.

Reply:
That's a really nice feature. I am updating!